packages/cli/src/credentials/credentials.service.ee.ts

```diff
@@ -93,4 +93,8 @@ export class EECredentialsService extends CredentialsService {
 
 		return credential;
 	}
+
+	static async getCredentials(filter: FindManyOptions<ICredentialsDb>): Promise<ICredentialsDb[]> {
+		return Db.collections.Credentials.find(filter);
+	}
```

Comment:
Perhaps as `getMany()` since the entity is clear from the context? And also maybe moved to the free version? Re: the latter, `getCredentials()` is only being used for the paid version right now, but simply fetching entities by a filter is basic service functionality, and likely to be duplicated when the free version needs it. We prolly need a good definition of what belongs in paid services.